martin/src/pg/errors.rs

```diff
@@ -69,4 +69,7 @@ pub enum PgError {
 
     #[error(r#"Unable to get tile {2:#} with {:?} params from {1}: {0}"#, query_to_json(.3.as_ref()))]
     GetTileWithQueryError(#[source] TokioPgError, String, TileCoord, Option<UrlQuery>),
+
+    #[error("Validation error: {0}")]
+    ValidationError(String),
```

Comment:
Lets call this `ConfigError`. Also, at least for now, all errors are `&'static str` - so we don't need to call `.into()` on them to convert str into an allocated String.